src/Microsoft.DotNet.Darc/DarcLib/Remote.cs

```diff
@@ -77,7 +77,6 @@ public Task<bool> BranchExistsAsync(string repoUri, string branch)
     /// <returns>List of pull request checks</returns>
     public async Task<IEnumerable<Check>> GetPullRequestChecksAsync(string pullRequestUrl)
     {
-        CheckForValidGitClient();
```

Comment:
I also removed this because it is an artifact from a time when DI was not used for this and there were several arguments passed to ctor and some could have been `null`. I had it sitting in my local so pushing it too